### ISSUE TITLE
[Usable - Make Code - Download microbit] Visually appearing 'Help' link looks as button on 'Download to your micro bit' pop-up window.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2371]
**User Experience:**
User will miss the functionality If any actionable control(link) display as button as they might consider that actionable control as button, so it would be confusing for users about the control functionality and will skip the relevant information of such controls.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Download" button and select it.
6. Navigate through all the controls appearing under "Download to your micro:bit" dialog.
7. Verify 'Help' link visually.

**Actual Result:**
Visually appearing 'Help' control display as a button instead of 'link' on 'Download to your micro bit' pop-up window, due to this user would consider that control button which will affect the functionality of that control.

**Expected Result:**
'Help' link should have underline or chevron to identify visually it as a link on 'Download to your micro bit' pop-up window, so that user could interact with link and get the relevant information.

**Note:**
This issue also exists with Chrome.

**Labels**
#A11yUsable, #HCL, #HCL-MakeCode-E2E, #Win10-Chrome, #Win10-Edge, #accessibility

![Usable_Download_Link looks as a button](https://user-images.githubusercontent.com/30456910/62621096-b0fa1f80-b938-11e9-8ce3-52901b670293.jpg)

**Comment by abchatra** [comment]
Won't fix. This doesn't have classification and unclear what rule this breaks. We have help with links to external documentation.

**Comment by BhuwanNaswa** [comment]
Labels
#A11yUsable, #HCL, #HCL-MakeCode-E2E, #Win10-Chrome, #Win10-Edge, #accessibility, #closed
